spec/features/send_test_message_spec.rb

```diff
@@ -14,8 +14,56 @@
     expect(page).to have_content("Test message sent")
   end
 
+  scenario "message sends successfully" do
+    create_message
+    create_employee
+    login_with_oauth
+    visit messages_path
+
+    page.find(".button-test").click
+    fill_in "Slack username", with: username_from_fixture
+    click_on "Send test"
+
+    expect(page).to have_content("Test message sent")
+  end
+
+  scenario "attempt to send test to Slack username that does not exist" do
+    create_message
+    login_with_oauth
+    visit messages_path
+
+    page.find(".button-test").click
+    fill_in "Slack username", with: "notreal"
+    click_on "Send test"
+
+    expect(page).to have_content("isn't in the Dolores system")
+  end
+
+  scenario "attempt to send test to employee missing Slack info" do
+    create_message
+    create(
+      :employee,
+      slack_username: username_from_fixture,
+      slack_channel_id: nil,
+      slack_user_id: nil,
+    )
+
+    login_with_oauth
+    visit messages_path
+
+    page.find(".button-test").click
+    fill_in "Slack username", with: username_from_fixture
+    click_on "Send test"
+
+    expect(page).to have_content("isn't up to date")
```

Comment:
note to self: we should put flash messages in I18n so they are easier to update and check for in tests